src/quicklist.c

```diff
@@ -69,6 +69,12 @@ int quicklistisSetPackedThreshold(size_t sz) {
  * 8k is a recommended / default size limit */
 #define SIZE_SAFETY_LIMIT 8192
 
+/* Maximum estimate of the listpack entry overhead.
+ * Although in the worst case(sz < 64), we will waste 5 bytes in one
+ * quicklistNode, but can avoid memory fragmentation due to listpack
+ * exceeding the limit size. */
```

Comment:
plain "fragmentation" usually refers to external fragmentation/
```suggestion
 * quicklistNode, but can avoid memory waste due to internal fragmentation
 * when the listpack exceeds the size limit by a few bytes (e.g. being 16388). */
```